src/index.md

```diff
@@ -1445,6 +1445,48 @@ Disadvantages of this model are that the list of backports in JBS will still lis
 [To the top](#){.boxheader}
 :::
 
+# Working With Pull Requests
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [Skara documentation on PR commands](https://wiki.openjdk.java.net/display/SKARA/Pull+Request+Commands)
+:::
+
+Once you have made a change that you want to integrate into an OpenJDK code base you need to create a _Pull Request_ (PR) on GitHub. This guide assumes that you have previous experience from using git and GitHub and won't go into details of how things work. Still, the aim is of course to provide a useful guide, so speak up if more details are needed.
+
```

Comment:
of how things work ->  of how those work 

so speak up if more details are needed.

"so send email to  guide dot dev at openjdk dot java net if ..." ?

Reply:
Fixed.